pkg/apis/machine/v1beta1/machine_webhook.go

```diff
@@ -84,6 +84,10 @@ var (
 )
 
 const (
+	DefaultWebhookName        = "machine-api-operator-webhook"
+	DefaultMutatingHookPath   = "/mutate-machine-openshift-io-v1beta1-machine"
```

Comment:
We should include the fact that these are for machines in here, eg
```suggestion
	DefaultMachineMutatingHookPath   = "/mutate-machine-openshift-io-v1beta1-machine"
```